checkstyle-tester/diff.groovy

```diff
@@ -161,6 +135,20 @@ def hasUnstagedChanges(gitRepo) {
     return hasUnstagedChanges
 }
 
+def runPlainCheckstyleReportsGeneration(cfg) {
+    generateCheckstyleReport(cfg.localGitRepo, cfg.patchBranch, cfg.patchCheckstyleCfg, cfg.listOfProjects, cfg.reportsDir)
+    generateSummaryIndexHtml(cfg.reportsDir)
```

Comment:
PR for patch-diff-report-util started at https://github.com/checkstyle/contribution/pull/216